atomic_physics/common.py

```diff
@@ -406,7 +406,7 @@ def setB(self, B: float):
 
             for M in set(self.M[lev]):
                 for Fidx, idx in np.ndenumerate(np.where(M == self.M[lev])):
-                    self.F[lev][idx] = F_list[M <= F_list][Fidx[1]]
+                    self.F[lev][idx] = F_list[abs(M) <= F_list][Fidx[1]]
```

Comment:
Wasn't that fixed in a separate PR?

Reply:
Yes - though this PR came from the same branch that had all the Ba isotopes as well so the change is here too (forgot to remove...).